storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -101,12 +101,19 @@ sakila/payment.ibd
 EOF
 
 # PXB-1832: Xbcloud does not exit when a piped command fails
-xbcloud --defaults-file=$topdir/xbcloud.cnf get 2>$topdir/pxb-1832.log | true
+xbcloud --defaults-file=$topdir/xbcloud.cnf get ${full_backup_name} 2>$topdir/pxb-1832.log | true
 if [ "${PIPESTATUS[0]}" == "0" ] ; then
     die 'xbcloud did not exit with error'
 fi
 
-if [ ! grep failed 2>$topdir/pxb-1832.log ] ; then
+if ! grep -q failed $topdir/pxb-1832.log ; then
+    die 'xbcloud did not exit with error'
+fi
+
+#PXB-2164 xbcloud doesn't return the error if the backup doesn't exist in s3 bucket
+xbcloud --defaults-file=$topdir/xbcloud.cnf get somedummyjunkbackup 2>$topdir/pxb-2164.log
```

Comment:
do u remove these *.log files somewhere?

Reply:
they are removed as part of topdir